docs/gallery.md

```diff
@@ -0,0 +1,306 @@
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        const p = await api.getPlugin("https://github.com/imjoy-team/imjoy-interactive-segmentation/blob/master/interactive-ml-demo.imjoy.html")
+        await p.run({})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+## Microscopy Control with Micro-Manager
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        const p = await api.getPlugin("https://gist.github.com/oeway/7a5313eb4fb7ed09a5f6d60adb561d59")
+        await p.run({})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+
+## HPA-UMAP
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        await api.createWindow({src: "https://github.com/imjoy-team/example-plugins/blob/master/imjoy-plugins/HPA-UMAP.imjoy.html", name: "HPA-UMAP"})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+## Image Annotator
+
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        await api.createWindow({src: "https://github.com/imjoy-team/example-plugins/blob/master/imjoy-plugins/ImageAnnotator.imjoy.html", name: "Image Annotator"})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+
+## HPA-Classification
+
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        await api.createWindow({src: "https://github.com/imjoy-team/example-plugins/blob/master/imjoy-plugins/HPA-Classification.imjoy.html", name: "HPA-Classification"})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+
+
+## Skin-Lesion-Analyzer
+
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        await api.createWindow({src: "https://github.com/imjoy-team/example-plugins/blob/master/imjoy-plugins/Skin-Lesion-Analyzer.imjoy.html", name: "Skin-Lesion-Analyzer"})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+## DeepBindScan
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        const p = await api.getPlugin("https://github.com/imjoy-team/example-plugins/blob/master/imjoy-plugins/DeepBindScan.imjoy.html")
+        await p.run({})
+    }
+}
+api.export(new ImJoyPlugin())
+```
+
+## CARE
+<!-- ImJoyPlugin: {"type": "web-worker", "editor_height": "400px", "hide_code_block": true} -->
+```js
+class ImJoyPlugin {
+    async setup() {}
+    async run(ctx) {
+        const imjoy = await api.createWindow({src: "https://imjoy.io/#/app?w=sandbox&flags=quite"})
```

Comment:
Should it be `quiet`?

Reply:
Yes, fixed here:https://github.com/imjoy-team/ImJoy/pull/422/files

Thanks!